cli/config/compose/stacks/metricbeat/docker-compose.yml

```diff
@@ -11,4 +11,4 @@ services:
       - ELASTIC_PASSWORD=p4ssw0rd
     image: "docker.elastic.co/elasticsearch/elasticsearch:${stackVersion}"
     ports:
-      - "9200:9200"
+      - "${elasticsearchPort:-9200}:9200"
```

Comment:
What is the difference between this `elasticsearch` service and the other `elasticsearch` one, defined in `cli/config/compose/services/elasticsearch/docker-compose.yml`?

Reply:
There are two abstractions:
- stacks, which is a preconfigured set of services that are the runtime dependencies for a specific project, like the metricbeat tests, or the stack-monitoring parity tests.
- services, which are isolated services that can be aggregated to running services or stacks, one by one.

The metricbeat stack only contains one runtime dependency, which is an ES instance, but it could involve a Kibana for UI verifications.

Another idea of stacks for stacks is to share configurations across teams

Reply:
Ah, I didn't realize this. Thanks for explaining!